clarifai/cli/model_templates.py

```diff
@@ -0,0 +1,182 @@
+    return '''from typing import Any
+
+from fastmcp import FastMCP  # use fastmcp v2 not the built in mcp
+from pydantic import Field
+
+from clarifai.runners.models.mcp_class import MCPModelClass
+
+# TODO: please fill in
+# Configure your FastMCP server
+server = FastMCP("my-mcp-server", instructions="", stateless_http=True)
+
+
+# TODO: please fill in
+# Add your tools, resources, and prompts here
+@server.tool("example_tool", description="An example tool")
+def example_tool(input_param: Any = Field(description="Example input parameter")):
+    """Example tool implementation."""
+    # TODO: please fill in
+    # Implement your tool logic here
+    return f"Processed: {input_param}"
+
+
+# Static resource example
+@server.resource("config://version")
+def get_version():
+    """Example static resource."""
+    # TODO: please fill in
+    # Return your resource data
+    return "1.0.0"
+
+
+@server.prompt()
+def example_prompt(text: str) -> str:
+    """Example prompt template."""
+    # TODO: please fill in
+    # Define your prompt template
+    return f"Process this text: {text}"
+
+
+class MyModel(MCPModelClass):
+    """A custom model implementation using MCPModelClass."""
+
+    def get_server(self) -> FastMCP:
+        """Return the FastMCP server instance."""
+        return server
+'''
+
+
+def get_openai_model_class_template() -> str:
+    """Return the template for an OpenAIModelClass-based model."""
+    return '''from openai import OpenAI
+
+from clarifai.runners.models.openai_class import OpenAIModelClass
+
+
+class MyModel(OpenAIModelClass):
+    """A custom model implementation using OpenAIModelClass."""
+    
+    # TODO: please fill in
+    # Configure your OpenAI-compatible client
+    client = OpenAI(
+        api_key="your-api-key-here",  # TODO: please fill in
+        base_url="https://api.openai.com/v1",  # TODO: please fill in - modify for your API endpoint
+    )
+    
+    # TODO: please fill in
+    # Specify the model name to use
+    model = "gpt-3.5-turbo"  # TODO: please fill in - replace with your model name
+    
+    def load_model(self):
+        """Optional: Add any additional model loading logic here."""
+        # TODO: please fill in (optional)
+        # Add any initialization logic if needed
+        pass
+'''
+
+
+def get_config_template(model_type_id: str = "text-to-text") -> str:
+    """Return the template for config.yaml."""
+    return f'''# Configuration file for your Clarifai model
+
+model:
+  id: "my-model"  # TODO: please fill in - replace with your model ID
+  user_id: "user_id"  # TODO: please fill in - replace with your user ID
+  app_id: "app_id"  # TODO: please fill in - replace with your app ID
+  model_type_id: "{model_type_id}"
+
+build_info:
+  python_version: "3.12"
+
+# TODO: please fill in - adjust compute requirements for your model
+inference_compute_info:
+  cpu_limit: "1"
+  cpu_memory: "1Gi"
+  num_accelerators: 0
```

Comment:
maybe include the fields needed for accelerators and the requests (not just limits) so users know about them by default.